.buildkite/scripts/run_models.sh

```diff
@@ -19,7 +19,6 @@ dbt deps
 dbt seed --target "$db" --full-refresh
 dbt run --target "$db" --full-refresh
 dbt test --target "$db"
-dbt run --target "$db"
+dbt run --vars '{employee_history_enabled: true}' --target "$db" 
 dbt test --target "$db"
```

Comment:
When running the test we need to turn on the variable. Otherwise the history tests will not run.

```suggestion
dbt test --vars '{employee_history_enabled: true}' --target "$db"
```

Reply:
Updated